samples/teamsfx-app/README.md

```diff
@@ -1,5 +1,36 @@
-## TeamsFx app using Microsoft Graph Toolkit components
+# Personal tab across Microsoft 365 using Microsoft Graph Toolkit components
 
-You can find the best TeamsFx sample app using Microsoft Graph Toolkit components on the TeamsFx repo. Click [here](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/graph-toolkit-contact-exporter) to be redirected.
+Personal tabs provide a great way to enhance the Microsoft Teams experience with the Microsoft Graph Toolkit. Using personal tabs, you can provide a user access to their application right within Teams, without the user having to leave the experience or sign in again. Now, personal tabs can light up within other Microsoft 365 applications. You can build and run your personal tabs in Teams, both Outlook desktop and web experiences, and also the Microsoft 365 app on the web.
 
-![Contact Exporter Sample](https://github.com/OfficeDev/TeamsFx-Samples/raw/dev/graph-toolkit-contact-exporter/images/overview.gif)
\ No newline at end of file
+![Personal tab demo](https://user-images.githubusercontent.com/11220663/167839153-0aef6adc-450e-4b8c-a28f-7d27005d1093.png)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/), supported versions: 16, 18
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/microsoft-365/dev-program)
+- [Set up your dev environment for extending Teams apps across Microsoft 365](https://aka.ms/teamsfx-m365-apps-prerequisites)
+  > Please note that after you enrolled your developer tenant in Office 365 Target Release, it may take couple days for the enrollment to take effect.
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
+
+## Getting Started
+
+Follow below instructions to get started with this application template for local debugging.
+
+### Test your application with Visual Studio Code
+
+1. Press `F5` or use the `Run and Debug Activity Panel` in Visual Studio Code.
+1. Select a target Microsoft 365 application where the personal tabs can run: `Debug in Teams`, `Debug in Outlook` or `Debug in the Microsoft 365 app` and click the `Run and Debug` green arrow button.
+
+### Test your application with TeamsFx CLI
+
+1. Executing the command `teamsfx provision --env local` in your project directory.
+1. Executing the command `teamsfx deploy --env local` in your project directory.
+1. Executing the command `teamsfx preview --env local --m365-host <m365-host>` in your project directory, where options for `m365-host` are `teams`, `outlook` or `office`.
```

Comment:
Resulting app has https error, do we need guidance on how to bypass?
Resulting app has errors at runtime: 
![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/7122716/2379ef25-2620-4e0c-a826-7dd17b40be9d)

Are we missing a file in the commit or a step to generate the file?

Reply:
Removed this reference.

Reply:
Added a note about the HTTPs issues while referencing teamsfx core help